meeting-notes/2019/Q1/2019-01-21--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,181 @@
+    - Created a rough roadmap for rolling out [Rendezvous](https://gist.github.com/jacobheun/13247de56e61e717c8a6d682406d01bf)
+      - Will work on improving that and will turn that into an Awesome Endeavor issue this week.
+    - We finalized a draft of the [rendezvous spec](https://github.com/libp2p/specs/blob/e1083c1f9d8f7afc0d65a43a12b05492f3873385/rendezvous/README.md), which will enable us to finish up the initial work on the rendezvous go and js implementations
+- Blocked: N/A
+- Next:
+  - Create `Awesome Endeavor` issue to track the release of Rendezvous. 
+    - Will include decommissioning plan for \*-star servers.
+  - Continue looking into js heap issue when exchanging files [libp2p#309](https://github.com/libp2p/js-libp2p/issues/309)
+  	- Will create an issue in ipfs/benchmark for this and refernce the slowness with go cat'ing from js
+  - Finalize libp2p daemon PR
+  - Create plan for completing needed work for multi address listening on transports and graceful listening configuration for Switch. Context at [interface-transport#41](https://github.com/libp2p/interface-transport/issues/41)
+
+@vmx
+- Done:
+  - Still working on getting the [new js-ipld API](https://github.com/ipld/js-ipld/pull/185) work with js-ipfs
+- Blocked:
+  - Not yet, but once the js-ipfs tests pass with the new js-ipld API: Using constants for multicodecs: https://github.com/multiformats/js-multicodec/pull/35
+- Next:
+  - Still working on getting the [new js-ipld API](https://github.com/ipld/js-ipld/pull/185) work with js-ipfs
+
+@hugomrdias (not attending)
+- Done:
+  - still working on making mplex work
+  - make pull-stream-to-stream return a proper readable-stream 3
+  - ci prototypes are passing 
+- Blocked:
+- Next:
+  - fix this https://github.com/ipfs/js-ipfs/issues/1827#issuecomment-455501157
+  - fix cors related issues in aegir test 
+  - hopefully this week Jacob can help me with mplex and im able to finish the bundle size PRs 
+
+
+@zcstarr(not attending)
+- Done:
+	- js-ipfs-repo (async/await) changes thx for pointers last week should be able to test changes soon 
+- Blocked:
+- Next:
+  - js-ipfs-repo changes
+
+@vasco-santos
+- Done:
+  - Libp2p team week
+    - Planning for 2019 and protocols discussion
+    - Research on local transports/web bluetooth for libp2p
+  - DHT PR rebased with newest `js-ipfs` release
+- Blocked:
+  - DHT tests
+    - [ipfs/js-ipfs#1827](https://github.com/ipfs/js-ipfs/issues/1827)
+- Next:
+  - Get [ipfs/js-ipfs#856](https://github.com/ipfs/js-ipfs/pull/856) ready to be merged
+  - Write pubsub benchmark test
+  - Meeting with ChainSafe in the context of collaborating for gossipsub in JS land [ChainSafeSystems/gossipsub-js](https://github.com/ChainSafeSystems/gossipsub-js/)
+  - Get [libp2p-daemon-client#1](https://github.com/libp2p/js-libp2p-daemon-client/pull/1) merged
+  - Continue `libp2p` interop work
+
+@achingbrain
+- Done:
+	- Adding support for IPFS to npm
+  - Package Managers kickoff (working group)
```

Comment:
Is there a repo for package managers WG? + we get a link that and to the kickoff meeting here?

Reply:
Not yet. Soon™ We are getting a "Roadmap" like doc ready